app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java

```diff
@@ -188,8 +194,8 @@ public void onDestroyView() {
         String desc = titleDesc.getString("Desc", "");
         Timber.d("Title: %s, Desc: %s", title, desc);
 
-        titleEdit.setText(title);
         descEdit.setText(desc);
+        titleEdit.setText(title);
```

Comment:
Yeah, this looks inconsistent. Title is above description both in the variable declaration and the UI.